packages/app-project/src/shared/components/SubjectSetPicker/SubjectSetPicker.stories.js

```diff
@@ -1,42 +1,34 @@
-import { Box } from 'grommet'
-
 import SubjectSetPicker from './'
-import { mockWorkflow } from './helpers'
-
-function DecoratedStory(Story) {
-  return (
-    <Box pad='small'>
-      <Story />
-    </Box>
-  )
-}
+import { SubjectSetPickerMock } from './SubjectSetPicker.mock'
+import { Grommet } from 'grommet'
+import { zooTheme } from '@zooniverse/grommet-theme'
 
 export default {
   title: 'Project App / Shared / Subject Set Picker',
   component: SubjectSetPicker,
-  decorators: [DecoratedStory],
   args: {
-    workflow: mockWorkflow
+    workflow: SubjectSetPickerMock
   }
 }
 
-export function Default({ workflow }) {
-  return (
+export const Default = ({ workflow }) => (
+  <Grommet theme={zooTheme}>
```

Comment:
I think all our stories are wrapped in Grommet by default.

Reply:
Moved from the Story to the Spec